src/app/loadout/loadout-ui/Sockets.m.scss

```diff
@@ -12,3 +12,9 @@
     --item-size: var(--item-icon-size);
   }
 }
+
+.automaticallyPicked {
+  :global(.item-img) {
+    border: 1px solid coral;
```

Comment:
I might suggest `$orange` to try and suggest that it's DIM that picked the mod. Or maybe a green.